src/components/Common/Header/LoginNav.tsx

```
@@ -2,24 +2,41 @@ import { useState } from "react"
 import styled from "@emotion/styled"
 import ConfirmRoundButton from "../Button/RoundButton"
 import { Color, Font } from "../../../styles"
+import { useNavigate } from "react-router-dom"
 
 const LoginNav = () => {
-  const [user, setUser] = useState(true);
+  const navigation = useNavigate();
+  const [isLoggedIn, setIsLoggedIn] = useState(true);
```

ESLint

[error] 'setIsLoggedIn' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)